addon/bootstrap.js

```diff
@@ -100,16 +103,24 @@ async function startup(addonData, reason) {
 
   log.debug(`info ${JSON.stringify(studyUtils.info())}`);
 
-  let value = HomepageStudy.reportValue(studyUtils.studyName);
+  let value = study.reportValue(studyConfig.studyName, studyConfig.isSimulation, studyConfig.rapporSimulatorPath);
+
+  if (studyConfig.isSimulation){
```

Comment:
Let's move this right below the previous line (no blank line) and change it to:

```
if (studyConfig.isSimulation || !value) {
  studyUtils.endStudy({reason: "ignored"});
  return;
}
```

Let's also remove the `if `block below this that specifically checks for `!value`.